sample_data/oereblex/plr119/contaminated_military_sites/document.json

```diff
@@ -0,0 +1,161 @@
+[
```

Comment:
I find it is confusing to have all this non-oereblex data in the oereblex test case. I understand it is only for those themes that are configured as standard. But it will be hard to remember that and in any case, we are already testing the standard data model in the non-oereblex case.

Suggestion: we remove all standard theme data in the Oereblex case (keep only the theme for which we provide Oereblex specific sample data).

Reply:
It was much easier to implement it this way (just use another path for the sample data in the Makefile) but I will look for a better solution

Reply:
Note: this has been solved by using symbolic links to the standard sample data